p2p/p2p-core/src/services.rs

```diff
@@ -15,21 +16,28 @@ pub enum PeerSyncRequest<N: NetworkZone> {
         current_cumulative_difficulty: u128,
         block_needed: Option<u64>,
     },
-    /// Add/update a peers core sync data to the sync state service.
+    /// Add/update a peers core sync data.
     IncomingCoreSyncData(InternalPeerID<N::Addr>, ConnectionHandle, CoreSyncData),
 }
 
+/// A response from the service that keeps track of peers sync states.
 pub enum PeerSyncResponse<N: NetworkZone> {
     /// The return value of [`PeerSyncRequest::PeersToSyncFrom`].
     PeersToSyncFrom(Vec<InternalPeerID<N::Addr>>),
     /// A generic ok response.
     Ok,
 }
```

Comment:
Is `Ok` more common? Might be worth defining it first.

Reply:
why?

Reply:
The reasoning is:

In large `match`es, if it isn't optimized into a jump table, [variants near the front are faster](https://github.com/lmammino/mega-match-vs-phf?tab=readme-ov-file#conclusions) (supposedly due to branch prediction). Regardless of speed difference, I have a mental model of variants being defined from `most likely` to `least likely`.

`PeerSyncResponse` is 2 variants so it _really_ doesn't matter, just a big nit, you can ignore this.

Reply:
I put the `Ok` last as it is a generic response and not very meaningful, I don't think these will be matched on much so I'm going to leave as is.